api/tests/prescription/campaign/unit/infrastructure/exports/campaigns/campaign-profiles-collection-result-line_test.js

```diff
@@ -48,6 +48,121 @@ describe('Unit | Serializer | CSV | campaign-profiles-collection-result-line', f
+          userId: 123,
+          participantFirstName: 'Juan',
+          participantLastName: 'Carlitos',
+          pixScore: 13,
+          additionalInfos: { hobby: 'genky' },
+        };
+
+        const csvExcpectedLine =
+          `"${organization.name}";` +
+          `${campaign.id};` +
+          `"${campaign.code}";` +
+          `"${campaign.name}";` +
+          `"${campaignParticipationResultData.participantLastName}";` +
+          `"${campaignParticipationResultData.participantFirstName}";` +
+          `"${campaignParticipationResultData.additionalInfos.hobby}";` +
+          '"Oui";' +
+          '2019-03-01;' +
+          '13;' +
+          '"Non";' +
+          '0;' +
+          '1;' +
+          '9;' +
+          '0;' +
+          '4' +
+          '\n';
+
+        //when
+        const line = new CampaignProfilesCollectionResultLine({
+          campaign,
+          organization,
+          campaignParticipationResult: campaignParticipationResultData,
+          competences,
+          placementProfile,
+          translate,
+          additionalHeaders: [{ columnName: 'hobby' }],
+        });
+
+        //then
+        expect(line.toCsvLine()).to.equal(csvExcpectedLine);
+      });
+
+      it('should not return addtional infos when nothing given', async function () {
+        //given
+        sinon.stub(PlacementProfile.prototype, 'isCertifiable').returns(true);
+        sinon.stub(PlacementProfile.prototype, 'getCertifiableCompetencesCount').returns(5);
+
+        campaign.id = 123;
+        campaign.code = 'ABCDEF';
+        campaign.name = 'test';
+        organization.name = 'Umbrella';
+
+        const campaignParticipationResultData = {
+          id: 1,
+          isShared: true,
+          isCompleted: true,
+          createdAt: new Date('2019-02-25T10:00:00Z'),
+          sharedAt: new Date('2019-03-01T23:04:05Z'),
+          userId: 123,
+          participantFirstName: 'Juan',
+          participantLastName: 'Carlitos',
+          pixScore: 13,
+          additionalInfos: { hobby: 'genky' },
+        };
+
+        const csvExcpectedLine =
+          `"${organization.name}";` +
+          `${campaign.id};` +
+          `"${campaign.code}";` +
+          `"${campaign.name}";` +
+          `"${campaignParticipationResultData.participantLastName}";` +
+          `"${campaignParticipationResultData.participantFirstName}";` +
+          '"Oui";' +
+          '2019-03-01;' +
+          '13;' +
+          '"Oui";' +
+          '5;' +
+          '1;' +
+          '9;' +
+          '0;' +
+          '4' +
+          '\n';
+
+        //when
+        const line = new CampaignProfilesCollectionResultLine({
+          campaign,
+          organization,
+          campaignParticipationResult: campaignParticipationResultData,
+          competences,
+          placementProfile,
+          translate,
+          additionalHeaders: [],
+        });
+
+        //then
+        expect(line.toCsvLine()).to.equal(csvExcpectedLine);
```

Comment:
**Suggestion:** le expect refleterait plus l'intention en disant "not include genky"